02-csv-vector-raster-plotting.Rmd

```diff
@@ -126,26 +125,26 @@ head(plot.location)
  * What is the coordinate reference system (CRS) for this data? 
  
 
-When we create a spatial points dataframe from a csv, there is no inherent
-Coordinate Reference System (CRS) associated with it. We can assign a CRS if we
+When we create a spatial points dataframe from a .csv, there is no inherent
+coordinate reference system (CRS) associated with it. We can assign a CRS if we
 know what the coordinate system the x,y values are in.
 
 In this case, the CRS for this data is UTM zone 18N (utmZone is 18N for all
 points in the header we just looked at). Luckily this is the same as our boundary
-shapefiles that we previously opened . 
+shapefiles that we previously opened. 
 
-If it is not still open in R, let's open up the shapefile `HarClip_UTMZ18".
+If it is not still open in R, let's open up the shapefile `HarClip_UTMZ18`.
 
 ```{r read-shp }
-#note: read ogr is preferred as it maintains prj info
+#note: readGR is preferred as it maintains the projection infomation
 aoiBoundary <- readOGR("boundaryFiles/HARV", "HarClip_UTMZ18")
 
 ```
 
-This is the shapefile that we want to match up with our csv file.  It is
-important to first confirm that the CRS of our CSV coordinates is the same as 
+This is the shapefile that we want to match up with our .csv file.  It is
```

Comment:
Changed throughout all lessons in Spatial Vector.  Will check in Temporal next time I work on that module.